esrally/rally.py

```diff
@@ -98,6 +98,9 @@ def supported_es_version(v):
                 raise argparse.ArgumentTypeError(f"must be at least {min_es_version} but was {v}")
         return v
 
+    def strip_whitespace(v):
+        return str(v).strip()
```

Comment:
nit: https://docs.python.org/3/library/argparse.html#type implies that `str` isn't needed. And I'm wondering if the function should not be called `revision` instead so that it reads like `type=revision`.

```suggestion
        return v.strip()
```

Reply:
I like `type=revision`. Thanks for the suggestion.